rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/Sw360ResourceServer.java

```diff
@@ -119,4 +139,26 @@ public FilterRegistrationBean<ForwardedHeaderFilter> forwardedHeaderFilter() {
         bean.setFilter(new ForwardedHeaderFilter());
         return bean;
     }
+
+    @Bean
+    public OpenAPI customOpenAPI() {
+        Server server = new Server();
+        server.setUrl(SERVER_PATH_URL + APPLICATION_NAME + REST_BASE_PATH);
+        server.setDescription("Current instance.");
+        return new OpenAPI()
+                .components(new Components()
+                        .addSecuritySchemes("tokenAuth",
+                                new SecurityScheme().type(SecurityScheme.Type.APIKEY).name("Authorization")
+                                        .in(SecurityScheme.In.HEADER)
+                                        .description("Enter the token with the `Token ` prefix, e.g. \"Token abcde12345\"."))
+                        .addSecuritySchemes("oauth",
+                                new SecurityScheme().type(SecurityScheme.Type.OAUTH2)
+                                        .flows(new OAuthFlows().password(new OAuthFlow()
+                                                .tokenUrl(SERVER_PATH_URL + "/authorization/oauth/token")
+                                                .refreshUrl(SERVER_PATH_URL + "/authorization/oauth/token"))
+                                        )))
+                .info(new Info().title("SW360 API").license(new License().name("EPL-2.0")
+                        .url("https://github.com/eclipse-sw360/sw360/blob/main/LICENSE")))
```

Comment:
Need help here. I want to read the version of SW360 (e.g. 17.0.0-SNAPSHOT) here and pass it as the version to the `Info` object. I am not sure from where or how to read it. It will be a great help if someone can guide me.

Reply:
@GMishx Maybe take a look on this:
https://www.baeldung.com/java-accessing-maven-properties

Reply:
Thanks for the suggestion. I added a new property called `rest.version` in main `pom.xml` to hold the version specific to REST API.